recipes/CommonVoice/ASR/transducer/hparams/train_fr.yaml

```diff
@@ -183,7 +183,7 @@ log_softmax: !new:speechbrain.nnet.activations.Softmax
    apply_log: True
 
 transducer_cost: !name:speechbrain.nnet.losses.transducer_loss
-   use_torchaudio: True
+   framework: torchaudio
```

Comment:
How did ither recipes got touched here?
Looks like you came across interface changes that require changes for CommonVoice, LibriSpeech & TIMIT as well?

Reply:
true